### PR TITLE
Storybook Upgrade

### PR DESCRIPTION
**Pull request by korgon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ node_modules
 yarn-error.log
 lerna-debug.log
 .DS_Store
+.nx
```
Comment (korgon) [inline .gitignore:5]: New cache location for NX.

File: .eslintignore [hunk 2]
```diff
@@ -4,4 +4,5 @@
 **/tests/**
 **/mocks/**
 **/__tests__/**
-**/__mocks__/**
\ No newline at end of file
+**/__mocks__/**
+**/jest.config.js
```
Comment (korgon) [inline .eslintignore:8]: Eslint not happy about new jest configs - just ignoring them in linting.

File: lerna.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
 	"version": "0.49.0",
 	"packages": ["packages/*"],
-	"useWorkspaces": true,
```
Comment (korgon) [inline lerna.json:4]: No longer needed in new version of Lerna.

File: packages/snap-preact-components/src/utilities/storybook.tsx
Comment (korgon) [inline packages/snap-preact-components/src/utilities/storybook.tsx:1]: This is a workaround to the issue seen where Storybook was crashing due to syntax highlighting in markdown files.

File: packages/snap-controller/jest.config.js [hunk 4]
```diff
@@ -1,5 +1,15 @@
 const rootConfig = require('../../jest.base.config.json');
+
+// const esModules = ['uuid', 'preact'].join('|');
+
```
Comment (chrisFrazier77) [inline packages/snap-controller/jest.config.js:4]: should we remove this?

**Review by chrisFrazier77 — APPROVED** [review]

**Review by dkonieczek — APPROVED** [review]

File: packages/snap-controller/jest.config.js (hunk 4 above)
Comment (korgon) [reply packages/snap-controller/jest.config.js:4]: Yes.

File: packages/snap-controller/jest.config.js [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
 module.exports = {
 	...rootConfig,
 	displayName: 'snap-controller',
+	moduleNameMapper: {
+		'^@testing-library/preact$': '<rootDir>/../../node_modules/@testing-library/preact/dist/cjs/index.js',
+		'^preact$': '<rootDir>/../../node_modules/preact/dist/preact.js',
+		'^preact/hooks$': '<rootDir>/../../node_modules/preact/hooks/dist/hooks.js',
+		'^preact/test-utils$': '<rootDir>/../../node_modules/preact/test-utils/dist/testUtils.js',
+	},
+	// transformIgnorePatterns: [`<rootDir>/node_modules/(?!(${esModules}))`],
```
[the remark this answers is not in the document]
Comment (korgon) [reply packages/snap-controller/jest.config.js:14]: Yup.

File: packages/snap-preact-components/src/documentation/About/About.stories.mdx [hunk 6]
```diff
@@ -15,15 +17,21 @@ import Readme from '../About/About.md';
...
 	<span class="subtitle">SNAP PREACT COMPONENTS</span>
 </div>
 
 ---
 
-<Readme />
\ No newline at end of file
+<Markdown  options={{
+	overrides: {
+		code: highlightedCode
```
[the remark this answers is not in the document]
Comment (korgon) [reply packages/snap-preact-components/src/documentation/About/About.stories.mdx:34]: Yea, it does look weird in Github. I'll format it locally and see if it resolves.

[the remark this answers is not in the document]
Comment (korgon) [reply packages/snap-preact-components/src/documentation/About/About.stories.mdx]: It is showing the same for theme and usage also... don't think it is an issue.

**Pull request merged**